app/javascript/modal.js

```diff
@@ -205,4 +202,14 @@ document.addEventListener("DOMContentLoaded", function() {
       }
     );
   }
-});
+}
+
+
+/*
+ * Setup the DOM event listeners on links requesting confirmation.
+ */
+document.addEventListener("DOMContentLoaded", detectModals);
```

Comment:
How does this work?

Reply:
Before we had a function that was being called when the `DOMContentLoaded` event occurred. I extracted that function and gave it name `detectModels` and added it on the window so we can call it again in the `create.js.erb`